qemu/tests/cfg/qmp_event_notification.cfg

```diff
@@ -50,6 +50,7 @@
             event_check = "RESUME"
         - qmp_rtc_change:
             no Windows
+            no arm64-pci, arm64-mmio
```

Comment:
How about  `no aarch64`?   It is more brief.   And current implement can work as expect too.

Reply:
Changed. This is better (I wasn't aware that arch type can be used as filter keyword).